src/docs/output-targets/custom-elements.md

```diff
@@ -96,9 +112,15 @@ The configs below provide examples of how to do this automatically with popular
 
 ## Distributing Custom Elements
 
-Your component library can be easily distributed on NPM, similar to how [`@ionic/core`](https://www.npmjs.com/package/@ionic/core) does it. From there, consumers of your library can decide how to import your library into their project. For the `dist-custom-elements`, the default import location would be `my-library/dist/components`, but this can get further configured within the `package.json` file.
+Your component library can be easily distributed on NPM, just like
```

Comment:
```suggestion
Your component library can be distributed on NPM, just like
```

nit: as a general rule of thumb I try to avoid words like "easily" or "just". "Easy" to someone may not necessarily be easy for someone else

Reply:
👍